core/schema.py

```diff
@@ -0,0 +1,9 @@
+import graphene
+
+import app.api.schema
```

Comment:
Or you could do this. `from app.api import schema as api_schema`, then `api_schema.Query`